data/migrations/dba_operator/e8e27f36f87d-databasemigration.yaml

```diff
@@ -0,0 +1,62 @@
+
+---
+apiVersion: dbaoperator.app-sre.redhat.com/v1alpha1
```

Comment:
I thought we were going to start putting migrations in their own, separate PRs to decouple from application logic in case we need to roll back.

Reply:
We can certainly do so; I created this PR before we decided to do so